radio/src/translations/se.h

```diff
@@ -344,8 +344,8 @@
 #define TR_NAME                         "Namn"
 #define TR_MODELNAME                    "Modellnamn"
 #define TR_PHASENAME                    "Lägesnamn"
-#define TR_MIXNAME                      TR("Mixer","Mixernamn")
-#define TR_INPUTNAME                    TR("Input","Inputnamn")
+#define TR_MIXNAME                      TR("Namn","Mixernamn")
+#define TR_INPUTNAME                    TR("Namn","Inputnamn")
```

Comment:
I'll let this go for now, but I'm not entirely convinced it is the right way to be doing this... i.e. might as well just use `TR_NAME` if going to lose the specificity of the translation string. I think the B&W UI needs a little rework around the inputs page as you have the input name (which gets used in the rest of the UI) and also a line name that gets shown in the inputs list... and that latter detail isn't very clear when it just lists it as "Input" and "Name"...